commands/bash/container/test-local/test-refresh.bats

```diff
@@ -2,9 +2,10 @@
 set -o pipefail
 
 @test "Test aladdin refresh command" {
-    # Call aladdin refresh 
+    # Call aladdin refresh
     $PY_MAIN refresh aladdin-test-server
     node_port=$(kubectl get svc aladdin-test-server -o json | jq -r '.spec.ports[] | select (.name == "http").nodePort')
+    local minikube_ip=$(kubectl config view -o jsonpath='{.clusters[?(@.name == "minikube")].cluster.server}' | sed -e "s/[^/]*\/\/\([^@]*@\)\?\([^:/]*\).*/\2/")
```

Comment:
yow.  Could we use a different separator than `/`, if our pattern has so many literal forward slashes in it?  All these escapes make this pretty hard to read.  Maybe '|' or '#' ?

Reply:
i've always been partial to `|` in these situations

Reply:
cleaned this up